src/server/api/schema.js

```diff
@@ -3541,7 +3555,13 @@ const rootResolvers = {
       { organizationId, cursor, campaignsFilter, role },
       { user }
     ) => {
-      await accessRequired(user, organizationId, "SUPERVOLUNTEER");
+      if (organizationId) {
+        await accessRequired(user, organizationId, "SUPERVOLUNTEER");
+      } else if (!user.is_superadmin) {
+        throw new ForbiddenError(
```

Comment:
Suggestion: Can this be simplified to
```suggestion
      if (organizationId) {
        await accessRequired(user, organizationId, "SUPERVOLUNTEER", true);
      } else {
        throw new ForbiddenError(
```

Also outside the scope of this PR but I wonder if we should create an issue to change `accessRequired` to default to `allowSuperadmin = true` There's a note from way back in [`EXPLANATION-development-guidelines.md`](https://github.com/politics-rewired/Spoke/blob/main/docs/EXPLANATION-development-guidelines.md?plain=1#L142) about the decision but I'm not sure that in the way that superadmin has evolved, we would ever call this without also intending to let superadmins perform whatever action?